.eslintrc.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  env: {
+    'node': true
+  },
+  extends: [
+    'standard'
+  ],  
+  rules: {
+    'array-bracket-spacing': [2, 'never'],
+    'block-scoped-var': 2,
+    'brace-style': [2, '1tbs'],
+    'camelcase': 1,
+    'computed-property-spacing': [2, 'never'],
+    'curly': 2,
+    'eol-last': 2,
+    'eqeqeq': [2, 'smart'],
+    'max-depth': [1, 3],
+    'max-len': [1, 80],
+    'max-statements': [1, 15],
+    'new-cap': 1,
+    'no-extend-native': 2,
+    'no-mixed-spaces-and-tabs': 2,
+    'no-trailing-spaces': 2,
+    'no-unused-vars': 1,
+    'no-use-before-define': [2, 'nofunc'],
+    'object-curly-spacing': [2, 'never'],
```

Comment:
Not sure about this one... recommended is never, but personally I would prefer to set it to _always_, so it requires spacing inside of braces (except {})

ie. 
`var obj = { 'foo': 'bar' };` instead of `var obj = {'foo': 'bar' };`

@JSKitty what do you think?

Reply:
An other one is for you to decide is "brace-style"

current:
```
try {
  somethingRisky();
} catch(e) {
  handleError();
}
```

changed to`'brace-style': [2, "stroustrup", { "allowSingleLine": true }],`
```
try {
  somethingRisky();
}
catch(e) {
  handleError();
}
```